packages/yoshi-flow-editor/src/cli.ts

```diff
@@ -1,3 +1,6 @@
+// We want to default yoshi.config.js. It will be removed when yoshi will be refactored to not use loadConfig from
+// multiple places outside of the methods.
```

Comment:
```suggestion
// We want to always load the same yoshi.config.js
```

I would add the "TODO" on yoshi config itself